tests/FakeItEasy.Tests/Expressions/CallExpressionParserTests.cs

```diff
@@ -176,7 +176,7 @@ private static MethodInfo GetPropertyGetter<T>(string propertyName)
         {
             return typeof(T)
                 .GetProperties(BindingFlags.Instance | BindingFlags.Public | BindingFlags.NonPublic)
-                .Where(x => x.Name.Equals(propertyName))
+                .Where(x => x.Name == propertyName)

```

Comment:
More readable IMHO, and doesn't incur a warning (equivalent to `Equals` with `StringComparison.Ordinal`)